INSTRUCTION.md

```diff
@@ -0,0 +1,19 @@
+# Instructions to Run and Stop the Application
+
+## Prerequisites
+- Docker and Docker Compose must be installed on your system.
+
+## Steps to Run the Application
+1. Clone the repository:
+   ```bash
+   git clone https://github.com/your-username/repository-name.git
+   cd repository-name
+   
+
+	2.	Build and start the containers:
+   
+docker-compose up --build
+
+3.	Access the application:
+	
+•	Visit http://localhost:8000 in your browser.
```

Comment:
If there are any additional configuration steps required before accessing the application (e.g., setting environment variables, configuring a database), make sure to include them in the instructions.